test/controllers/home.js

```diff
@@ -0,0 +1,16 @@
+const supertest = require('supertest');
+const tape = require('tape');
+const app = require('../../src/app');
+
+
+tape('testing the home route technology', (t) => {
+  supertest(app)
+    .get('/home')
+    .expect(200)
+    .expect('x-powered-by', 'Express')
+    .end((err, res) => {
+      t.error(err);
+      t.equal(Object.values(res.headers).indexOf('Express') > -1, true, 'headers should include the express technology');
```

Comment:
So you already said `.expect('x-powered-by', 'Express')` but then you make another equal to check that the headers actually contain `Express`? What's so important about `'x-powered-by', 'Express'` That you want to make sure its there so bad?